packages/react-native/Libraries/Image/Image.d.ts

```diff
@@ -208,11 +212,15 @@ export interface ImagePropsBase
    * should be used if the image is smaller than the view. It should also be used if the
    * image is slightly bigger than the view.
    *
+   * - `none`: No sampling is performed and the image is displayed in its full resolution.
+   * This should only be used in rare circumstances because it is considered unsafe
+   * as Android will throw a runtime exception when trying to render images that consume too much memory.
+   *
    * More details about `resize` and `scale` can be found at http://frescolib.org/docs/resizing-rotating.html.
    *
    * @platform android
    */
-  resizeMethod?: 'auto' | 'resize' | 'scale' | undefined;
+  resizeMethod?: 'auto' | 'resize' | 'scale' | 'none' | undefined;
```

Comment:
Since this interface already inherits from `ImagePropsAndroid`, could you remove this override from `ImagePropsBase`?

Reply:
@Abbondanzo    unable to be covered after testing. all need to be added to the ts definition.
<img width="604" alt="image" src="https://github.com/user-attachments/assets/606e8509-e7a5-48fd-9ddc-1e4f77a93766">

Reply:
The full override should be removed since it's redundant (i.e. lines 200 thru 223) and the `resizeMethod` on line 78 should be treated as the single source of truth. It would also be helpful if this could consolidate both docstrings too, replacing lines 61 thru 78 with:

```
  /**
   * The mechanism that should be used to resize the image when the image's dimensions
   * differ from the image view's dimensions. Defaults to `auto`.
   *
   * - `auto`: Use heuristics to pick between `resize` and `scale`.
   *
   * - `resize`: A software operation which changes the encoded image in memory before it
   * gets decoded. This should be used instead of `scale` when the image is much larger
   * than the view.
   *
   * - `scale`: The image gets drawn downscaled or upscaled. Compared to `resize`, `scale` is
   * faster (usually hardware accelerated) and produces higher quality images. This
   * should be used if the image is smaller than the view. It should also be used if the
   * image is slightly bigger than the view.
   *
   * - `none`: No sampling is performed and the image is displayed in its full resolution. This
   * should only be used in rare circumstances because it is considered unsafe as Android will
   * throw a runtime exception when trying to render images that consume too much memory.
   *
   * More details about `resize` and `scale` can be found at http://frescolib.org/docs/resizing-rotating.html.
   *
   * @platform android
   */
  resizeMethod?: 'auto' | 'resize' | 'scale' | 'none' | undefined;
```